draft-ietf-taps-impl.md

```diff
@@ -327,31 +327,29 @@ Branching for derived endpoints is the final step, and may have multiple layers
 
 Implementations should sort the branches of the tree of connection options in order of their preference rank. 
 Leaf nodes on branches with higher rankings represent connection attempts that will be raced first.
-Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Protocol and Path Selection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
+Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Selection and Connection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
 
 In addition to the properties provided by the application, an implementation may include additional criteria such as cached performance estimates, see {{performance-caches}}, or system policy, see {{role-of-system-policy}}, in the ranking.
-Two examples of how the Protocol and Path Selection Properties may be used to sort branches are provided below:
+Two examples of how the Selection and Connection Properties may be used to sort branches are provided below:
 
-* Interface Type:
+* Selection Property "Interface Instance or Type":
 If the application specifies an interface type to be preferred or avoided, implementations should rank paths accordingly.
```

Comment:
Selection Property should not be needed here.

Reply:
I don't see the problem but removed "Selection Property" here and "Connection Property" in the next item, that should be a safe way out either way.